qc_baselib/__init__.py

```diff
@@ -3,7 +3,7 @@
 # Public License, v. 2.0. If a copy of the MPL was not distributed
 # with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-__version__ = "0.1.0"
+__version__ = "1.0.0rc1"
```

Comment:
```suggestion
__version__ = "v1.0.0-rc1"
```

Reply:
Actually, according to SemVer, this should be `v1.0.0-rc.1`. I updated the version here to `v1.0.0-rc.1`.